docs/testing/integration_tests.rst

```diff
@@ -2,4 +2,50 @@
 Integration Tests
 *****************
 
-- To be completed
+
+Configuration
+=============
+
+The main configuration file to look at is ``/tests/protractor.conf.js``.
+It configures our ``browserName``.
+
+In ``browserName`` we specify the browser that will be used to launch the tests.
+It can be set to ``phantomjs``, ``firefox`` or ``chrome``.
+
+You can find more information about this in the
+`protractor referenceConf.js
+<https://github.com/angular/protractor/blob/master/docs/referenceConf.js>`_ documentation.
+
+All spec files should be placed in ``/tests/integration/specs`` and all page
+object files should be in ``/tests/integration/pages``. So, the file organisation
+structure is:
+
+```
+/tests/integration/specs
```

Comment:
use tree structure similar to https://github.com/aldryn/aldryn-boilerplate-bootstrap3/blob/master/docs/structure/general.rst#private